dcat/index.html

```diff
@@ -3323,30 +3328,264 @@ <h2>Version types</h2>
+<li>Those indicating whether a version is replaced/superseded by another one.</li>
+<li>Those indicating whether a version is backward compatible with another one.</li>
+</ol>
+
+<section id="version-history">
+
+<h2>Version chains and hierarchies</h2>
+
+<p>DCAT defines specific properties for describing version history, aligned with the corresponding [[?PAV]] ones:</p>
+
+<ul>
+<li><code>dcat:previousVersion</code> (equivalent to <a data-cite="?PAV#d4e459"><code>pav:previousVersion</code></a>)</li>
+<li><p><code>dcat:hasVersion</code> (equivalent to <a data-cite="?PAV#d4e395"><code>pav:hasVersion</code></a>), plus the following additional properties:</p>
+<ul>
+<li><code>dcat:isVersionOf</code> (inverse of <code>dcat:hasVersion</code>);</li>
+<li><code>dcat:currentVersion</code> (equivalent to <a data-cite="?PAV#d4e359"><code>pav:currentVersion</code></a>, and subproperty of <code>dcat:hasVersion</code>)</li>
+<li><code>dcat:lastVersion</code> (subproperty of <code>dcat:hasVersion</code>)</li>
+</ul>
+</li>
+</ul>
+
+<aside class="ednote">
+<p>The previous draft [[VOCAB-DCAT-3-20201217]] was making use of similar properties from [[DCTERMS]] and [[VOCAB-ADMS]], namely:</p>
+<ul>
+<li><code>dct:hasVersion</code> / <code>dct:isVersionOf</code>, for version hierarchies, and</li>
+<li><code>adms:prev</code>, <code>adms:next</code>, <code>adms:last</code> for both versions chains and dataset series.</li>
+</ul>
+<p>Their definitions and use do not however fit completely with similar properties in [[PAV]]. E.g.:</p>
+<ul>
+<li><a href="https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#http://purl.org/dc/terms/hasVersion"><code>dct:hasVersion</code></a> is defined as <q>A related resource that is a version, edition, or adaptation of the described resource</q>, thus using the notion of version in its wider sense, whereas [[PAV]] is specifically describing versions as the result of a revision.</li>
+<li><a data-cite="VOCAB-ADMS#adms-last"><code>adms:last</code></a> is defined as <q>A link to the current or latest version of the Asset</q>, whereas in [[PAV]] the current and latest versions of a resource may be different, and link to by using different properties.</li>
+</ul>
+<p>Moreover, although the definitions of the [[VOCAB-ADMS]] properties states explicitly that they are meant to be used for "versions", they are subproperties of the corresponding terms in [[XHTML-VOCAB]], where they are used to link resources in a collection (e.g., the set of pages of a Web site). E.g., the definition of <a data-cite="XHTML-VOCAB#last"><code>xhv:last</code></a> reads as follows: <q>last refers to the last resource in a collection of resources.</q>.</p>
+<p>Based on that, in this draft, [[PAV]]-equivalent properties are used for versions, whereas the [[VOCAB-ADMS]] ones are used for dataset series (<a href="#dataset-series"></a>). Properties <code>dct:hasVersion</code> / <code>dct:isVersionOf</code> are not used.</p>
```

Comment:
Will you include `equivalentProperty` axioms in the RDF?

Reply:
BTW - I agree with not using PAV directly. It was nicely designed but has rarely been used, and thus would be an unnecessary and out-of-date dependency. I was on a call with Stian Soiland-Reyes and he expressed some surprise that there was continued interest in PAV. So I think we can assume that PAV itself is no longer a going concern.

Reply:
That's very useful, @dr-shorthair .

About your first question, equivalence in the RDF can be stated either formally or informally. What would you suggest?